proxy/http/HttpTunnel.cc

```diff
@@ -956,8 +956,8 @@ HttpTunnel::producer_run(HttpTunnelProducer *p)
 
     // (note that since we are not dechunking POST, this is the chunked size if chunked)
     if (p->buffer_start->read_avail() > HttpConfig::m_master.post_copy_size) {
-      Warning("http_redirect, [HttpTunnel::producer_handler] post exceeds buffer limit, buffer_avail=%" PRId64 " limit=%" PRId64 "",
-              p->buffer_start->read_avail(), HttpConfig::m_master.post_copy_size);
+      Debug("http_redirect", "[HttpTunnel::producer_handler] post exceeds buffer limit, buffer_avail=%" PRId64 " limit=%" PRId64 "",
```

Comment:
If this happened, the post request will fail. Are we sure this is something we can ignore?

Reply:
@zizhong Is the post failing completely or will it just fail to do a follow redirect?

Reply:
In common cases, it fails only for the redirect. When post buffer(PR#2335) enabled, it will fail for the original too.

Reply:
That is not good for the common case, with buffering enabled.  What change made it fail when post buffer is enabled?   I don't believe this is how it worked before.

Reply:
that is #2335, which utilizes the same buffer logic.